schema/metaschema/oscal-catalog-metaschema.xml

```diff
@@ -0,0 +1,422 @@
+
+  <define-field as="mixed" group-as="citations" name="citation" show-docs="xml json">
+    <flag name="id" datatype="ID"/>
+    <flag name="href" datatype="anyURI"/>
+    <formal-name>Citation</formal-name>
+    <description>Citation of a resource</description>
+    <remarks>
+      <p>Semantics of any contents within citation are implicit, not encoded. To mitigate problems establishin
+        matches with bibliograpical databases, it is recommended a canonical
+      identifier such as a DOI be provided for any citation.</p>
+    </remarks>
+    <example>
+      <o:citation>Some <o:strong>citation</o:strong> of some sort</o:citation>
+    </example>
+  </define-field>
+
+  <define-flag name="id" datatype="ID" show-docs="xml json">
+    <formal-name>ID / identifier</formal-name>
+    <description>Unique identifier</description>
+    <remarks>
+      <p>No mechanism is proposed to ensure that <code>id</code> values do not collide across
+        different catalogs. Use profiling without <q>merge</q> to detect such clashes.</p>
+    </remarks>
+    <example>
+      <o:citation id="xyz2">Some <o:strong>citation</o:strong> of some sort</o:citation>
+    </example>
+  </define-flag>
+
+  <define-flag name="class" datatype="NMTOKEN" show-docs="xml json">
+    <formal-name>Class</formal-name>
+    <description>Identifies the property or object within the control; a semantic hint</description>
+    <remarks>
+      <p>Overloading this attribute with more than one value is permitted, but not recommended.</p>
+    </remarks>
+  </define-flag>
+
+  <define-flag name="href" datatype="anyURI" show-docs="xml json">
+    <formal-name>hypertext reference</formal-name>
+    <description>A link to a document or document fragment (actual, nominal or projected)</description>
+    <remarks>
+      <p>Note in particular that <code>href</code> targets include elements (locations) in
+        representations of documents that exist only by virtue of dynamic application, such as (for
+        example) the results of profile resolution.</p>
+    </remarks>
+  </define-flag>
+
+
+  <define-flag name="test" datatype="string" show-docs="xml json">
+    <formal-name>Constraint test</formal-name>
+    <description>A formal (executable) expression of a constraint</description>
+  </define-flag>
+
+  <define-flag name="rel" datatype="NCName" show-docs="xml json">
+    <formal-name>Relation</formal-name>
+    <description>Purpose of the link</description>
+  </define-flag>
+
+  <define-flag name="how-many" datatype="string" show-docs="xml json">
+    <formal-name>Cardinality</formal-name>
+    <description>When selecting, a requirement such as one or more</description>
+  </define-flag>
+  
+  <define-flag name="depends-on" datatype="IDREF" show-docs="xml json">
+    <formal-name>Depends on</formal-name>
+    <description>Another parameter invoking this one</description>
+  </define-flag>
+  
+  <define-flag name="model-version" datatype="NCName" show-docs="xml json">
```

Comment:
@wendellpiez can you clarify the intent of the `model-version` flag? Is it designed to address https://github.com/usnistgov/OSCAL/issues/57 in that it sets the version of the schema?